pkgs/tools/package-management/nix/common.nix

```diff
@@ -251,6 +252,9 @@ self = stdenv.mkDerivation {
 
     tests = {
       nixi686 = pkgsi686Linux.nixVersions.${self_attribute_name};
+      # Basic smoke test that needs to pass when upgrading nix.
+      # Note that this test does only test the nixVersions.stable attribute.
+      misc = nixosTests.nix-misc.default;
```

Comment:
Looks better than my approach, but I believe we also need to update the nixos test in lix?